### PR TITLE
yubico-authenticator 6.4.0

### PR DESCRIPTION
**Pull request by unshare**
Re-targeted to use GitHub releases

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [ ] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --cask --new <cask>` worked successfully.
- [ ] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Review by SMillerDev — APPROVED** [review]

**Review by chenrui333 — APPROVED** [review]

File: Casks/y/yubico-authenticator.rb [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
+      verified: "github.com/Yubico/yubioath-flutter/releases/download/"
   name "Yubico Authenticator"
   desc "Application for generating TOTP and HOTP codes"
   homepage "https://developers.yubico.com/yubioath-flutter/"
 
   livecheck do
-    url "https://developers.yubico.com/yubioath-flutter/Releases/"
-    regex(/href=.*?yubico[._-]authenticator[._-]v?(\d+(?:\.\d+)+)[._-]mac\.dmg/i)
+    url :url
+    strategy :github_latest
```
Comment (p-linnane) [inline Casks/y/yubico-authenticator.rb:13]: If upstream doesn't use GitHub Prereleases, we shouldn't need this. The default `:git` strategy can take care of it.

Comment (unshare) [reply Casks/y/yubico-authenticator.rb:13]: The upstream apparently doesn't use GitHub Prereleases, but yet it is GitHub-hosted, and maintainers may start doing prereleases at any moment as they see fit.
As prereleases are discouraged for Homebrew casks, I believe your suggestion not only doesn't apply, but also should be considered harmful.

Comment (p-linnane) [reply Casks/y/yubico-authenticator.rb:13]: While I appreciate the concern, the `:github_latest` strategy has rate-limiting implications, so we only use it where it's confirmed to be necessary.

Comment (p-linnane) [reply Casks/y/yubico-authenticator.rb]: @unshare Thank you. You can remove the entire `livecheck` block. GitHub URLs use the `:git` strategy implicitly.

**Review by p-linnane — APPROVED** [review]
Thank you as always @unshare!

**Pull request merged**